test/test_proxy_tensor.py

```diff
@@ -115,24 +128,24 @@ def _create_new_input(x):
     if x.dtype != torch.float:
         return x + 1
     if x.is_leaf:
-        return torch.rand_like(x, requires_grad=True)
+        return torch.rand_like(x, requires_grad=x.requires_grad)
     else:
         return torch.rand_like(x)
 
-class TestProxyTensor(TestCase):
-    tracing_mode = "real"
-
+class TestGenericProxyTensor(TestCase):
+    # WARNING: if any of your inputs are index tensors, DO NOT use this
+    # function
```

Comment:
why is that ?

Reply:
cuz it generates random integers which are probably out of bounds lol